src/lbaf/Model/lbsBlock.py

```diff
@@ -54,10 +54,10 @@ def __init__(
             o_ids = set()
 
         # Block index
-        if not isinstance(b_id, int) or isinstance(b_id, bool):
-            raise TypeError(
-                f"b_id: incorrect type {type(b_id)}")
-        self.__index = b_id
+        # if not isinstance(b_id, int) or isinstance(b_id, bool):
```

Comment:
Why is this commented out? Either uncomment or remove.

Reply:
Fixed (removed)